### ISSUE TITLE
Slow?

### ISSUE THREAD

**Issue by rdbende** [rdbende/Sun-Valley-ttk-theme#83]
Many people say that sv-ttk is really slow on their system. I'm not experiencing any lag, but it might be a good idea to use gif for _some_ widget images.

**Comment by rdbende** [comment]
Or it would be interesting how the SVG version performs on Windows. I wanna convert this to SVG anyways.

**Comment by TotallyNotSethP** [comment]
Any new info on this? I really wanna use sv-ttk for my Windows app

**Comment by rdbende** [comment]
I'm not really working on this project any more, only some occasional commits. I also have no idea how it could be made any faster, other that telling people not to use Windows :P
This theme is probably as fast as it gets, even if that is quite slow on Windows. However I can say that it is one of the fastest tkinter themes, and it is multiple times faster than customtkinter

**Comment by Andrew-J-Larson** [comment]
I've found that using certain versions of packages with it (e.g. `pywinstyles`) can affect the speed, that's maybe something to look into.

**Comment by rdbende** [comment]
In what way? Does it make things faster?

**Comment by Andrew-J-Larson** [comment]
Resizing the window becomes much slower, at least for that package (when using the windows acrylic theming).

But that hooks into windows api's, so I'm not sure it's something you really can fix speed on.

**Comment by rdbende** [comment]
Yeah. I mean, if `pywinstyles` makes the window laggier, then it's probably an issue with that package, and not with `sv-ttk`. It could also be that those Windows APIs are only optimized for Microsoft's own GUI toolkits.
